dirsrvtests/tests/suites/webui/__init__.py

```diff
@@ -0,0 +1,62 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+
+import pytest
+
+from lib389.utils import *
+from lib389.topologies import topology_st
+
+pytest.importorskip('playwright')
+
+def remove_instance_through_lib(topology):
+    log.info('Check and remove instance before starting tests')
+    if topology.standalone.exists():
+        topology.standalone.delete()
+
+
+def remove_instance_through_webui(topology, page):
+    frame = page.frame('cockpit1:localhost/389-console')
+
+    log.info('Check if instance exist')
+    if topology.standalone.exists():
+        log.info('Delete instance')
+        frame.wait_for_selector('#ds-action')
+        frame.click('#ds-action')
+        frame.click('#remove-ds')
+        frame.check('text=", I am sure." >> input[type="checkbox"]')
+        frame.click('//button[normalize-space(.)=\'Remove Instance\']')
+        frame.wait_for_selector("#no-inst-create-btn")
+        log.info('Instance deleted')
+    #frame.wait_for_selector("#no-inst-create-btn")
+
+    #assert frame.is_visible("#no-inst-create-btn")
+
+
+def setup_login(page):
+    page.set_viewport_size({"width": 1920, "height": 1080})
+    page.goto("http://localhost:9090/")
+
+    # We are at login page
+    page.fill('#login-user-input', 'root')
+    page.fill('#login-password-input', 'redhat')
+    page.click("#login-button")
+    page.wait_for_selector('text=Red Hat Directory Server')
```

Comment:
I've added missing dependencies. Test now fails here. On Fedora it's "389 Directory Server".
It would be also nice to capture screenshot on failure, so it would be easier to investigate such failures. See example here https://github.com/microsoft/playwright-pytest#screenshot-on-test-failure

Reply:
And `DEBUG=pw:api` would be useful too for stack traces from playwright.

Reply:
Ahh, I knew I wanted to include screenshots :D thanks for reminder.
I had to install slugify and unicode-slugify to make the paths work. Would this dependency be ok with us or do we want to modify it without slugify?
You mean to add DEBUG=pw:api to pytest.yml?

Reply:
> I had to install slugify and unicode-slugify to make the paths work. Would this dependency be ok with us or do we want to modify it without slugify?

Are both of these needed or just `python-slugify`?

> You mean to add DEBUG=pw:api to pytest.yml?

Correct.

Reply:
Just python-slugify should be enough.